oeq-ts-rest-api/src/BatchOperationResponse.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * The Apereo Foundation licenses this file to you under the Apache License,
+ * Version 2.0, (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Responses for batch operations
+ * such as batch create, update and delete
+ */
+export interface BatchOperationResponse {
+  id: string;
+  status: number;
+  message: string;
```

Comment:
Ah! Sorry Yinzi, I think your message above resulted in me skipping these. You've missed TSDoc on all three.

Should just be able to use the ones on server side, or if you need some help it's be along the lines of:

```suggestion
  /** The ID of the item the response belongs to. */
  id: string;
  /** A HTTP status code indicating what the result was. */
  status: number;
  /** A human readable message about the result. */
  message: string;
```

But in writing that for you, it made me wonder. What is `id` in the case of batch adding multiple items and one fails?

Reply:
@edalex-ian 
It returns an empty string.
```
[
  {
    "id": "abc63ff9-6ed6-47fe-a004-c92c133b68de",
    "status": 200,
    "message": "A new filter has been created. ID: abc63ff9-6ed6-47fe-a004-c92c133b68de"
  },
  {
    "id": "",
    "status": 400,
    "message": "Invalid MIMETypes found : aaabbb"
  }
]
```

Reply:
Okay, goodoh, that looks okay. Thanks. :+1: